autoflip_calculators/BUILD

```diff
@@ -0,0 +1,555 @@
+    deps = [":face_to_region_calculator_proto"],
+)
+
+cc_test(
+    name = "face_to_region_calculator_test",
+    srcs = ["face_to_region_calculator_test.cc"],
+    linkstatic = 1,
+    deps = [
+        ":face_to_region_calculator",
+        ":face_to_region_calculator_cc_proto",
+        "//mediapipe/examples/desktop/autoflip:autoflip_messages_cc_proto",
+        "//mediapipe/framework:calculator_framework",
+        "//mediapipe/framework:calculator_runner",
+        "//mediapipe/framework/formats:detection_cc_proto",
+        "//mediapipe/framework/formats:image_frame",
+        "//mediapipe/framework/formats:image_frame_opencv",
+        "//mediapipe/framework/formats:location_data_cc_proto",
+        "//mediapipe/framework/port:gtest_main",
+        "//mediapipe/framework/port:parse_text_proto",
+        "//mediapipe/framework/port:ret_check",
+        "//mediapipe/framework/port:status",
+        "@com_google_absl//absl/strings",
+    ],
+)
+
+proto_library(
+    name = "localization_to_region_calculator_proto",
+    srcs = ["localization_to_region_calculator.proto"],
+    deps = [
+        "//mediapipe/framework:calculator_proto",
+    ],
+)
+
+mediapipe_cc_proto_library(
+    name = "localization_to_region_calculator_cc_proto",
+    srcs = ["localization_to_region_calculator.proto"],
+    cc_deps = ["//mediapipe/framework:calculator_cc_proto"],
+    visibility = ["//mediapipe/examples:__subpackages__"],
+    deps = [":localization_to_region_calculator_proto"],
+)
+
+cc_library(
+    name = "localization_to_region_calculator",
+    srcs = ["localization_to_region_calculator.cc"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":localization_to_region_calculator_cc_proto",
+        "//mediapipe/examples/desktop/autoflip:autoflip_messages_cc_proto",
+        "//mediapipe/framework:calculator_framework",
+        "//mediapipe/framework/formats:detection_cc_proto",
+        "//mediapipe/framework/formats:location_data_cc_proto",
+        "//mediapipe/framework/port:ret_check",
+        "//mediapipe/framework/port:status",
+        "@com_google_absl//absl/memory",
+    ],
+    alwayslink = 1,
+)
+
+cc_test(
+    name = "localization_to_region_calculator_test",
+    srcs = ["localization_to_region_calculator_test.cc"],
+    linkstatic = 1,
+    deps = [
+        ":localization_to_region_calculator",
+        ":localization_to_region_calculator_cc_proto",
+        "//mediapipe/examples/desktop/autoflip:autoflip_messages_cc_proto",
+        "//mediapipe/framework:calculator_framework",
+        "//mediapipe/framework:calculator_runner",
+        "//mediapipe/framework/formats:detection_cc_proto",
+        "//mediapipe/framework/formats:location_data_cc_proto",
+        "//mediapipe/framework/port:gtest_main",
+        "//mediapipe/framework/port:parse_text_proto",
+        "//mediapipe/framework/port:ret_check",
+        "//mediapipe/framework/port:status",
+        "@com_google_absl//absl/strings",
+    ],
+)
+
+cc_library(
```

Comment:
Can you only include the build rules for the .cc files in this folder please?  So just text detection and it's test/protos.